themes/default/views/bundles/authority_references_list.php

```diff
@@ -38,8 +38,6 @@
 	$va_references				= $this->getVar('references');
 	$va_search_strings			= $this->getVar('search_strings');
 	$vn_max_items				= caGetOption('maxReferencesToDisplay', $va_settings, 100);
-
-	print caEditorBundleShowHideControl($this->request, $vs_id_prefix);
```

Comment:
These other views are just removing the show/hide button.  Pretty self-explanatory I guess.  I originally changed the method itself so that it just returned an empty string, but decided rather than changing core code I could keep it in the theme.  (I used `git rebase -i` to remove the commit before I put up the PR, so you won't see it here).